docs/cli/vast-export-json.md

```diff
@@ -1,2 +1,7 @@
 The JSON export format renders events in newline-delimited JSON (aka.
 [JSONL](https://en.wikipedia.org/wiki/JSON_streaming#Line-delimited_JSON)).
+
+By default, durations render as human-readable strings with up to two decimal
+places and a unit, e.g., `2.31s` or `24.1ms`. The option `--numeric-durations`
+causes VAST to render durations as numbers instead, representing the time in
+nanoseconds since the UNIX epoch.
```

Comment:
This sounds wrong to me. If the value is already a duration we have a start point already, replacing that with epoch doesn't make sense.

Reply:
Eh, brain fart moment, that's how we'd render timestamps. You saw nothing!